js/script.js

```diff
@@ -0,0 +1,24 @@
+const navOpen = document.querySelector('.open');
+const mobileNav = document.querySelector('.mobile-nav');
+const closeMenu = document.querySelector('.close');
+
+navOpen.addEventListener('click', () => {
+  mobileNav.style.display = 'flex';
+  document.body.style.overflow = 'hidden';
+});
+
+closeMenu.addEventListener('click', () => {
+  mobileNav.style.display = 'none';
+  document.body.style.overflow = 'auto';
+});
+
+window.addEventListener('scroll', () => {
+  const navbar = document.querySelector('.navbar');
+  if (window.scrollY > 50) {
+    navbar.style.backgroundcolor = '#fff';
+    navbar.style.boxShadow = '0 0px 10px 0 #00000033';
+  } else {
+    navbar.style.backgroundcolor = 'transparent';
+    navbar.style.boxShadow = 'none';
+  }
+});
```

Comment:
- [ ] Please use `DRY` principles in your project your project is not `DRY`. The `DRY (don't repeat yourself)` principle is a best practice in software development that recommends software engineers to do something once, and only once. Check below :point_down:  my suggestions :slightly_smiling_face: 

```suggestion
const getElement = (selector) => document.querySelector(selector);

const mobileNav = getElement('.mobile-nav');
const navbar = getElement('.navbar');
const navOpen = getElement('.open');
const closeMenu = getElement('.close');

const toggleMobileNav = (display, overflow) => {
  mobileNav.style.display = display;
  document.body.style.overflow = overflow;
};

const setNavbarStyles = (backgroundColor, boxShadow) => {
  navbar.style.backgroundColor = backgroundColor;
  navbar.style.boxShadow = boxShadow;
};

navOpen.addEventListener('click', () => {
  toggleMobileNav('flex', 'hidden');
});

closeMenu.addEventListener('click', () => {
  toggleMobileNav('none', 'auto');
});

window.addEventListener('scroll', () => {
  if (window.scrollY > 50) {
    setNavbarStyles('#fff', '0 0px 10px 0 #00000033');
  } else {
    setNavbarStyles('transparent', 'none');
  }
});
```

In the modified code:

1. The getElement function is introduced to simplify the process of selecting DOM elements by their class names.
2. The `toggleMobileNav` function is created to handle the common behaviour of setting the display and overflow properties of `mobileNav` and `document.body`.
3. The `setNavbarStyles` function is introduced to handle setting the `backgroundColor` and `boxShadow` properties of the navbar.
4. By using these functions, repetitive code is eliminated, and the code becomes more modular and easier to maintain.
Overall, this refactoring makes the code DRYer by reducing redundancy and promoting code reuse.